hello-world-python/README.md

```diff
@@ -0,0 +1,110 @@
+# Conclave Hello World Python Sample
```

Comment:
The contents in this README page would be better placed in Conclave docs website. Otherwise, the documentation for Conclave ends up in multiple places which can confuse people and lead to duplicated documents. 

I will leave this decision to Shams and you.

Reply:
ok, I agree so I will update the readme and just add references to the documentation.